src/index.ts

```diff
@@ -40,18 +36,45 @@ export {
   H6,
 } from "./components/Typography/Heading";
 export { IconButton } from "./components/IconButton/IconButton";
-export { Label } from "./components/Form/Label";
 export { Link } from "./components/Link/Link";
 export { MenuItem } from "./components/MenuItem/MenuItem";
-export { Message } from "./components/Form/Message";
-export { PasswordControl } from "./components/Form/Controls/Password";
-export { Radio } from "./components/Radio/Radio";
-export { Root } from "./components/Form/Root";
-export { Submit } from "./components/Form/Submit";
 export { Search } from "./components/Search/Search";
-export { Toggle } from "./components/Toggle/Toggle";
 export { Tooltip } from "./components/Tooltip/Tooltip";
-export { ValidityState } from "./components/Form/ValidityState";
+
+export {
+  TextControl,
+  /* Legacy export name */
+  TextControl as Control,
```

Comment:
I checked with EC and ESS and no one is using the form right now, other than MAS. @germain-gg: should I get rid of the "legacy" export names? (`Checkbox`, `Radio` and `Toggle` below as well)